src/Raven.Server/Documents/Revisions/RevisionsBinCleanMergedCommand.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using Raven.Server.Documents.TransactionMerger.Commands;
+using Raven.Server.ServerWide.Context;
+using Voron;
+
+namespace Raven.Server.Documents.Revisions;
+
+public partial class RevisionsStorage
```

Comment:
if this is a partial, then file should be called RevisionsStorage.RevisionsBinCleanMergedCommand .cs

Reply:
Done